src/components/WorkspacesGettingStarted/Card.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import '@testing-library/jest-dom';
+import { WorkspacesGettingStartedCard } from './index';
+
+jest.mock('../../components/HelpTopicLink', () => ({
+  HelpTopicLink: jest.fn(() => <span>mock help topic link</span>),
+}));
+
+describe('WorkspacesGettingStartedCard', () => {
+  it('should have an image', () => {
+    render(<WorkspacesGettingStartedCard />);
+    expect(screen.getByRole('img')).toBeVisible();
+  });
+
+  it('should have a title', () => {
+    render(<WorkspacesGettingStartedCard />);
+    expect(screen.getByText('Get started with Workspaces')).toBeVisible();
+  });
+
+  it('should not be dismissable', () => {
+    render(<WorkspacesGettingStartedCard />);
+    expect(screen.queryByRole('button')).toBeNull();
```

Comment:
In these cases, I'm a fan of use `.not.toBeInTheDocument()` or `.not.toBeVisible()` to make it clear what you are testing.